ArduCopter/Parameters.cpp

```diff
@@ -1168,12 +1168,48 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
 
     // ID 60 is reserved for the SHIP_OPS
 
+    // extend to a new group
+    AP_SUBGROUPEXTENSION("", 61, ParametersG2, var_info2),
+
     // ID 62 is reserved for the SHOW_... parameters from the Skybrush fork at
     // https://github.com/skybrush-io/ardupilot
 
     AP_GROUPEND
 };
 
+/*
+  extension to g2 parameters
+ */
+const AP_Param::GroupInfo ParametersG2::var_info2[] = {
+
+    // @Param: PLDP_THRESH
+    // @DisplayName: Payload Place thrust ratio threshold
+    // @Description: Ratio of vertical thrust during decent below which payload touchdown will trigger.
+    // @Range: 0.5 0.9
+    // @User: Standard
+    AP_GROUPINFO("PLDP_THRESH", 1, ParametersG2, pldp_throttle_placed_fraction, 0.9),
+
+    // @Param: PLDP_RNG_MIN
+    // @DisplayName: Payload Place minimum range finder altitude
+    // @Description: Minimum range finder altitude in m to trigger payload touchdown, set to zero to dissable.
```

Comment:
"dissable" should be "disable"

Reply:
I knew I would get something wrong there :)